code/datums/sims.dm

```diff
@@ -442,12 +442,7 @@
 	var/list/motives = list()
 	var/list/datum/simsHolder/simsHolders = list()
 	var/list/datum/simsMotive/simsMotives = list()
-
-#ifdef RP_MODE
 	var/provide_plumbobs = 0
-#else
-	var/provide_plumbobs = 1
-#endif
 
 	New()
 		..()
```

Comment:
if globally disabling plumbobs like this, `toggle_sims_mode` verb should re-enable them